addons/reload/CfgVehicles.hpp

```diff
@@ -6,7 +6,7 @@ class CfgVehicles {
                 class GVAR(LinkBelt) {
                     displayName = CSTRING(LinkBelt);
                     distance = 2.0;
-                    condition = QUOTE([ARR_2(_player, _target)] call FUNC(canLinkBelt));
+                    condition = QUOTE(([ARR_2(_player, _target)] call FUNC(canLinkBelt) > 0));
```

Comment:
Is this meant to be:
`QUOTE(([ARR_2(_player, _target)] call FUNC(canLinkBelt)) > 0)`

Otherwise the extra set of brackets aren't doing anything :stuck_out_tongue: